src/hotspot/share/oops/method.hpp

```diff
@@ -308,7 +308,7 @@ class Method : public Metadata {
     return _method_data;
   }
 
-  void set_method_data(MethodData* data);
+  void clear_method_data();
```

Comment:
This should be private now I think.

Reply:
Makes sense - done. I also inlined its definition into `method.hpp`.